src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -11,7 +11,9 @@ excerpt: "xk6-browser brings browser automation and end-to-end testing to k6 whi
 import launcher from 'k6/x/browser';
 
 export default function () {
-  const browser = launcher.launch('chromium', { headless: false });
+  const browser = launcher.launch('chromium', {
+    headless: false,
+  });
```

Comment:
WDYT about moving this example to the "Example" section, and replacing the example that's currently there? The one there isn't very useful, and we could merge the options demo with this one.

The main reason is because we should first explain what xk6-browser is and how to use/build it, before showing any examples. Otherwise people might try to run the example without reading the rest of the required steps.

BTW, remember to change it to use the new API from https://github.com/grafana/xk6-browser/pull/462.

Reply:
Sure, i'll make that change.

I'll amend the example in a different PR to conform to the API change in https://github.com/grafana/xk6-browser/pull/462.

Reply:
Partially resolved in 43f0d9825beaf62ff70739bc52a0859cf60d1524